### ISSUE TITLE
t/10metrics_*.t failures on freetype 2.9.1 + bleed

### ISSUE THREAD

**Issue by dwarring** [p6-pdf/Font-FreeType-p6#2]
```
 LD_LIBRARY_PATH=/home/david/git/freetype2/objs/.libs/ prove -e'perl6 -I .' t/10metrics_postscript.t
t/10metrics_postscript.t .. 25/53 # Failed test 'right bearing of glyph 'A''
# at t/10metrics_postscript.t line 111
# expected: '50'
#      got: '20'
# Failed test 'width of glyph 'A''
# at t/10metrics_postscript.t line 113
# expected: '1409'
#      got: '1439'
# Failed test 'right bearing of glyph '_''
# at t/10metrics_postscript.t line 111
# expected: '4'
#      got: '-17'
# Failed test 'width of glyph '_''
# at t/10metrics_postscript.t line 113
# expected: '1037'
#      got: '1058'
# Failed test 'left bearing of glyph '`''
# at t/10metrics_postscript.t line 109
# expected: '116'
#      got: '118'
# Failed test 'right bearing of glyph '`''
# at t/10metrics_postscript.t line 111
# expected: '245'
#      got: '235'
# Failed test 'width of glyph '`''
# at t/10metrics_postscript.t line 113
# expected: '321'
#      got: '329'
# Failed test 'left bearing of glyph 'g''
# at t/10metrics_postscript.t line 109
# expected: '56'
#      got: '57'
# Failed test 'right bearing of glyph 'g''
# at t/10metrics_postscript.t line 111
# expected: '56'
#      got: '36'
# Failed test 'width of glyph 'g''
# at t/10metrics_postscript.t line 113
# expected: '912'
#      got: '931'
# Failed test 'left bearing of glyph '|''
# at t/10metrics_postscript.t line 109
# expected: '160'
#      got: '163'
# Failed test 'right bearing of glyph '|''
# at t/10metrics_postscript.t line 111
# expected: '169'
#      got: '164'
# Failed test 'width of glyph '|''
# at t/10metrics_postscript.t line 113
# expected: '81'
#      got: '83'
# Looks like you failed 13 tests of 53
t/10metrics_postscript.t .. Dubious, test returned 13 (wstat 3328, 0xd00)
Failed 13/53 subtests 

Test Summary Report
-------------------
t/10metrics_postscript.t (Wstat: 3328 Tests: 53 Failed: 13)
  Failed tests:  31-32, 36-37, 40-42, 45-47, 50-52
  Non-zero exit status: 13
``

**Comment by dwarring** [comment]
Differences in font metrics. It looks as if the metrics width was wrong, but has been fixed.

This module was expecting the old incorrect width. These  should be updated to reflect the freetype 2.9.1+ corrections and skipped on earlier freetype versions.

**Comment by dwarring** [comment]
10metrics_oldstandart.t also has similar failures:
```
10metrics_oldstandart.t 
t/10metrics_oldstandart.t .. 1/27 # Failed test 'text height'
# at t/10metrics_oldstandart.t line 49
# expected: '1482'
#      got: '1236'
# Failed test 'ascender'
# at t/10metrics_oldstandart.t line 50
# expected: '952'
#      got: '762'
# Failed test 'descender'
# at t/10metrics_oldstandart.t line 51
# expected: '-294'
#      got: '-238'
# Looks like you failed 3 tests of 27
```

**Comment by dwarring** [comment]
Tests fixed for FreeType2 2.9.1+ and skipped for earlier versions
